client/src/components/DataFiles/DataFilesModals/DataFilesDownloadMessageModal.js

```diff
@@ -16,27 +105,85 @@ const DataFilesDownloadMessageModal = () => {
   };
 
   return (
-    <Modal isOpen={isOpen} toggle={toggle} size="md" className="dataFilesModal">
+    <Modal
+      isOpen={isOpen}
+      onOpened={onOpened}
+      onClosed={onClosed}
+      toggle={toggle}
+      className="dataFilesModal"
+    >
+      {/* { console.log(`operationStatus =`) }
+      { console.log(operationStatus) } */}
       <ModalHeader toggle={toggle} charCode="&#xe912;">
-        Download
+        Download Folder
       </ModalHeader>
-      <ModalBody>
-        <p styleName="title">Folders must be compressed before download.</p>
-        <ol>
-          <li>Select the folder(s).</li>
-          <li>
-            Press the &quot;
-            <Icon name="compress">↩</Icon> Compress&quot; icon above the table.
-          </li>
-          <li>Complete and submit the form.</li>
-          <li>After the job finishes, you may download the compressed file.</li>
-        </ol>
-      </ModalBody>
-      <ModalFooter>
-        <Button type="button" className="data-files-btn" onClick={toggle}>
-          Close
-        </Button>
-      </ModalFooter>
+      <Formik
+        innerRef={formRef}
+        initialValues={initialValues}
+        validationSchema={validationSchema}
+        onSubmit={compressCallback}
+      >
+        {({ setFieldValue, values, isValid }) => {
+          const handleSelectChange = e => {
+            setFieldValue('filetype', e.target.value);
+          };
+          const formDisabled = status === 'RUNNING' || status === 'SUCCESS';
+          const buttonDisabled =
+            formDisabled || !isValid || values.filenameDisplay === '';
+          return (
+            <Form>
+              <ModalBody>
+                <SectionMessage type="warning">
+                  Folders must be compressed before downloading
+                </SectionMessage>
+                <FormField
+                  label="Compressed File Name"
+                  name="filenameDisplay"
+                  disabled={formDisabled}
+                  addonType="append"
+                  addon={
+                    <InputGroupAddon addonType="append">
```

Comment:
Can you add the [style that Garrett added in a recent PR](https://github.com/TACC/Core-Portal/blob/main/client/src/components/DataFiles/DataFilesModals/DataFilesCompressModal.js#L130) to this `InputGroupAddon`?

Reply:
I added `styleName="input-field"` to `InputGroupAddon`.  I also removed a couple of commmented out console.logs that I left in accidentally.  I apologize for the error.